producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/submitquery/SubmitQueryConfigOne.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.moj.sdt.producers.comx.config.submitquery;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.moj.sdt.domain.SubmitQueryRequest;
+
+@Configuration
+@EnableAutoConfiguration
+public class SubmitQueryConfigOne {
+
+    @Bean("uk.gov.moj.sdt.domain.api.ISubmitQueryRequestMCOLDefence1")
+    public SubmitQueryRequest submitQueryRequest() {
+        SubmitQueryRequest submitQueryRequest = new SubmitQueryRequest();
+        submitQueryRequest.setResultCount(6);
+        submitQueryRequest.setStatus("Ok");
+        String response = """
+            <![CDATA[
+                              <sub:mcolDefenceDetail xmlns:sub="http://ws.sdt.moj.gov.uk/2013/sdt/SubmitQueryResponseSchema" xmlns:mquer="http://ws.sdt.moj.gov.uk/2013/mcol/QuerySchema">
```

Comment:
Can reduce the level of indentation of the contents of CDATA to make it more readable, e.g.
```
<![CDATA[
  <sub:mcolDefenceDetail xmlns:sub="http://ws.sdt.moj.gov.uk/2013/sdt/SubmitQueryResponseSchema" ...
```

Reply:
done